server/dal/src/database/photo.rs

```diff
@@ -182,7 +216,7 @@ impl<'a> Photo<'a> {
     /// # Errors
     ///
     /// If a database error occurs
-    pub async fn is_quality_created(&self, quality: PhotoQuality) -> DbResult<bool> {
+    pub async fn is_quality_created(&self, quality: &PhotoQuality) -> DbResult<bool> {
```

Comment:
Quality created fields can be removed from the database and replaced by checking if the amazon URL for the quality exists. Remove redundancy and prevent the information from getting out of sync

Reply:
Don't entirely agree. Checking if the URL exists is way more expensive (I think) than checking the database

Reply:
Isn't it negligible? PostgreSQL should be optimized enough to do this (depending on how often you perform this action, but in this case it should be way less hondreds of times per second I think)